packages/web-components/src/components/quote/quote.ts

```diff
@@ -14,6 +14,7 @@ import StableSelectorMixin from '../../globals/mixins/stable-selector';
 import { QUOTE_TYPES } from './defs';
 import '../horizontal-rule/horizontal-rule';
 import { carbonElement as customElement } from '../../internal/vendor/@carbon/web-components/globals/decorators/carbon-element';
+import { LocaleAPI } from '../../internal/vendor/@carbon/ibmdotcom-services/services/Locale';
```

Comment:
Found a bug here for downstream adopters. This compiles to:

```
import { LocaleAPI } from "../../internal/vendor/@carbon/ibmdotcom-services/services/Locale.js";
```
Which does not exist and throws errors with at least some bundler configurations. This adopter is using Webpack:

```
ERROR in ./node_modules/@carbon/ibmdotcom-web-components/es/components/quote/quote.js 31:0-96
Module not found: Error: Can't resolve '../../internal/vendor/@carbon/ibmdotcom-services/services/Locale.js' in '/Users/m4olivei/projects/ibm-cms-cloud-carbon-v2/ui.frontend.idl/node_modules/@carbon/ibmdotcom-web-components/es/components/quote'
```

It's also reproducible with a simple vite configuration: https://stackblitz.com/github/m4olivei/carbon-demo/tree/quote-test?file=components.js

Will file an issue and fix.

Reply:
https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/12034